operator/pkg/helmreconciler/reconciler.go

```diff
@@ -581,3 +587,100 @@ func (h *HelmReconciler) networkName() string {
 	}
 	return nw
 }
+
+func ProcessDefaultWebhook(client kube.Client, iop *istioV1Alpha1.IstioOperator, ns string) (processed bool, err error) {
+	// Detect whether previous installation exists prior to performing the installation.
+	exists := revtag.PreviousInstallExists(context.Background(), client.Kube())
+	rev := iop.Spec.Revision
+	isDefaultInstallation := rev == "" && iop.Spec.Components.Pilot != nil && iop.Spec.Components.Pilot.Enabled.Value
+	operatorManageWebhooks := operatorManageWebhooks(iop)
+	if !operatorManageWebhooks && (!exists || isDefaultInstallation) {
+		if rev == "" {
+			rev = revtag.DefaultRevisionName
+		}
+		autoInjectNamespaces := validateEnableNamespacesByDefault(iop)
+
+		o := &revtag.GenerateOptions{
+			Tag:                  revtag.DefaultRevisionName,
+			Revision:             rev,
+			Overwrite:            true,
+			AutoInjectNamespaces: autoInjectNamespaces,
+		}
+		// If tag cannot be created could be remote cluster install, don't fail out.
+		tagManifests, err := revtag.Generate(context.Background(), client, o, ns)
+		if err == nil {
+			if err = applyManifests(client, tagManifests); err != nil {
+				return false, err
+			}
+		}
+		processed = true
+	}
+	return processed, nil
+}
+
+func applyManifests(kubeClient kube.Client, manifests string) error {
+	yamls := strings.Split(manifests, helm.YAMLSeparator)
+	for _, yml := range yamls {
+		if strings.TrimSpace(yml) == "" {
+			continue
+		}
+
+		obj := &unstructured.Unstructured{}
+
+		if err := yaml.Unmarshal([]byte(yml), obj); err != nil {
+			return fmt.Errorf("failed to unmarshal YAML: %w", err)
+		}
+		var ogvr schema.GroupVersionResource
+		if obj.GetKind() == name.MutatingWebhookConfigurationStr {
+			ogvr = gvr.MutatingWebhookConfiguration
+		} else if obj.GetKind() == name.ValidatingWebhookConfigurationStr {
+			ogvr = gvr.ValidatingWebhookConfiguration
+		}
+
+		_, err := kubeClient.Dynamic().Resource(ogvr).Namespace(obj.GetNamespace()).Patch(
+			context.TODO(), obj.GetName(), types.ApplyPatchType, []byte(yml), metav1.PatchOptions{
+				Force:        nil,
+				FieldManager: "install",
```

Comment:
For consistency with other objects we may want to use https://github.com/istio/istio/blob/d0705cf0ed5591cc26c08001f3faab0a880aec48/operator/pkg/helmreconciler/apply.go#L36?